torchvision/prototype/transforms/functional/_dispatch.py

```diff
@@ -0,0 +1,448 @@
+
+
+@resize.implements(features.BoundingBox)
+def _resize_bounding_box(input: features.BoundingBox, *, size: List[int], **_: Any) -> features.BoundingBox:
+    intermediate_format = BoundingBoxFormat.XYXY
+    converted_input = F.convert_bounding_box_format(input, old_format=input.format, new_format=intermediate_format)
+
+    output = F.resize_bounding_box(converted_input, old_image_size=input.image_size, new_image_size=size)
+    output = F.convert_bounding_box_format(output, old_format=intermediate_format, new_format=input.format)
+
+    return features.BoundingBox.new_like(input, output, image_size="size")
+
+
+@resize.implements(features.SegmentationMask)
+def _resize_segmentation_mask(
+    input: features.SegmentationMask,
+    *,
+    size: List[int],
+    interpolation: InterpolationMode = InterpolationMode.NEAREST,
+    max_size: Optional[int] = None,
+    antialias: Optional[bool] = None,
+) -> features.SegmentationMask:
+    output = F.resize_segmentation_mask(
+        input, size=size, interpolation=interpolation, max_size=max_size, antialias=antialias
+    )
+
+    return features.SegmentationMask.new_like(input, output)
+
+
+@Dispatcher
+def center_crop(input: T, *, output_size: List[int]) -> T:
+    """ADDME"""
+    pass
+
+
+@center_crop.implements(features.Image, pil_kernel=_F.center_crop)
+def _center_crop_image(input: features.Image, *, output_size: List[int]) -> features.Image:
+    output = F.center_crop(input, output_size=output_size)
+
+    return features.Image.new_like(input, output)
+
+
+@Dispatcher
+def normalize(input: T, *, mean: List[float], std: List[float], inplace: bool = False) -> T:
+    """ADDME"""
+    pass
+
+
+@normalize.implements(features.Image)
+def _normalize_image(
+    input: features.Image, *, mean: List[float], std: List[float], inplace: bool = False
+) -> features.Image:
+    output = F.normalize(input, mean=mean, std=std, inplace=inplace)
+
+    return features.Image.new_like(input, output, color_space=ColorSpace.OTHER)
+
+
+@Dispatcher
+def resized_crop(
+    input: T,
+    *,
+    top: int,
+    left: int,
+    height: int,
+    width: int,
+    size: List[int],
+    interpolation: InterpolationMode = InterpolationMode.BILINEAR,
+) -> T:
+    """ADDME"""
+    pass
+
+
+@resized_crop.implements(features.Image, pil_kernel=_F.resized_crop)
+def _resized_crop_image(
+    input: features.Image,
+    *,
+    top: int,
+    left: int,
+    height: int,
+    width: int,
+    size: List[int],
+    interpolation: InterpolationMode = InterpolationMode.BILINEAR,
+) -> features.Image:
+    output = F.resized_crop(
+        input, top=top, left=left, height=height, width=width, size=size, interpolation=interpolation
+    )
+
+    return features.Image.new_like(input, output)
+
+
+@Dispatcher
+def erase(input: T, *, i: int, j: int, h: int, w: int, v: torch.Tensor, inplace: bool = False) -> T:
+    """ADDME"""
+    pass
+
+
+@erase.implements(features.Image, pil_kernel=_F.erase)
```

Comment:
nit: erase does not work on PIL images
https://github.com/pytorch/vision/blob/8aad0e0a64c65b4e16855764288f0bc2ad2e14c0/torchvision/transforms/functional.py#L1276-L1277

Reply:
Good catch, thanks!